next/components/forms/steps/Summary/StepButtonGroup.tsx

```diff
@@ -0,0 +1,44 @@
+import LeftIcon from '@assets/images/chevron-left.svg'
+import ArrowRightIcon from '@assets/images/forms/arrow-right.svg'
+import { useTranslation } from 'next-i18next'
+
+import Button from '../../simple-components/Button'
+
+interface StepButtonGroupProps {
+  stepIndex: number
+  isFinalStep: boolean
+  previous: () => void
+  skip: () => void
+  submitStep: () => void
+  submitForm: () => void
+}
+
+const StepButtonGroup = (props: StepButtonGroupProps) => {
+  const { stepIndex, isFinalStep, previous, skip, submitStep, submitForm } = props
+  const [t] = useTranslation('forms')
```

Comment:
it probably does not matter, but we use the `{}` syntax  (`const { t } = useTranslation('forms')` ) everywhere else, so let's do it here as well